### PR TITLE
Remove unneeded concurrency

### PR DESCRIPTION
**Pull request by lionello**

### CODE REVIEW

**Review by edwardrf — APPROVED** [review]
LGTM

File: src/cmd/cli/command/commands.go [hunk 1]
```diff
@@ -859,58 +831,92 @@ var composeUpCmd = &cobra.Command{
...
+				return err
 			}
-
-			wg.Wait() // Wait for tail ctrl + c
 		}
-		cancelTail()
-		wg.Wait() // Wait for tail to finish
 
-		printEndpoints(serviceInfos)
+		printEndpoints(deploy.Services)
```
Comment (nullfunc) [inline src/cmd/cli/command/commands.go:919]: The service states aren't updated in deploy.Services so won't be showing the proper states.

Comment (lionello) [reply src/cmd/cli/command/commands.go:919]: ah, so yeah that kind of mutation is not great™ so if we want to show those states we'll need a different solution

Comment (lionello) [reply src/cmd/cli/command/commands.go]: I saw the new fix.

**Review by nullfunc — APPROVED** [review]

**Pull request merged**